resources/metadata/cloudFoundryCreateService.yaml

```diff
@@ -143,7 +143,6 @@ spec:
         aliases:
           - name: cloudFoundry/serviceManifest
           - name: cfServiceManifest
-        default: 'service-manifest.yml'
       - name: manifestVariables
```

Comment:
The default still needs to work after the switch to golang

Reply:
added default back again and added fileExists boolean check for default manifest